modules/apps/headless/headless-asset-library/headless-asset-library-impl/src/main/java/com/liferay/headless/asset/library/internal/resource/v1_0/AssetLibraryResourceImpl.java

```diff
@@ -453,6 +511,34 @@ private Map<String, Boolean> _getDepotAppCustomizationMap(
 		return depotAppCustomizationMap;
 	}
 
+	private List<Long> _getEligibleGroupIds(Group group) throws Exception {
+		List<Long> ids = new ArrayList<>();
```

Comment:
Can you change this variable type for `long[]` like we have in other parts of code and also rename this to `groupIds`?